guides/v2.3/graphql/queries/products.md

```diff
@@ -513,6 +513,92 @@ query {
 }
 ```
 
+### Return minimum and maximum prices and discount information
+
+In the following example, a catalog price rule that provides a 10% discount on all fitness equipment is in effect. The product queried, `24-WG080`, is the Sprite Yoga Companion Kit bundle product. This product has two user-selected option that cause the price to vary. If you choose to query a product this is not a composite (bundle, group, or configurable) product, the minimum and maximum prices are the same.
+
+**Request**
+
+```graphql
+{
+  products(filter: {sku: {eq: "24-WG080"}}, sort: {name: ASC}) {
+    items {
+      name
+      sku
+      price_range {
+        minimum_price {
+          regular_price {
```

Comment:
regular_price and final_price also have a currency field. Maybe it would be good to include that in the example?